main.go

```diff
@@ -1244,7 +1248,8 @@ func (tf *transformer) prefillObjectMaps(files []*ast.File) {
 				if obj == nil {
 					continue // not found; skip
 				}
-				tf.cannotObfuscateNames[obj] = true
+				// TODO(mvdan): it seems like removing this doesn't break any tests.
+				recordAsNotObfuscated(obj)
```

Comment:
this really puzzles me, btw. it's almost like we can now support obfuscating cgo names using `//export`, or somehow we're not obfuscating them even without this call.